src/components/Slider/components/DotGroup.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Dot } from 'pure-react-carousel';
+import styled from 'styled-components';
+import { PURPLE_500, GRAY_700 } from 'styles/colors';
+
+interface DotGroupProps {
+  totalSlides: number;
+}
+
+const DotGroup = ({ totalSlides }: DotGroupProps) => (
+  <Wrapper>
+    {new Array(totalSlides).map((_, i) => (
```

Comment:
```suggestion
    {new Array(totalSlides).fill(0).map((_, i) => (
```
Иначе map не сработает, т.к. массив пустой.